### ISSUE TITLE
Lake Water Quality conversion fails with assertion error in xarray Zarr backend

### ISSUE THREAD

**Issue by pont-us** [bcdev/nc2zarr#14]
Conversion using the attached file produces "AssertionError: We should never get here. Function logic must be wrong." (see traceback)

[config.yaml.zip](https://github.com/bcdev/nc2zarr/files/6011221/config.yaml.zip)
[traceback.txt](https://github.com/bcdev/nc2zarr/files/6011223/traceback.txt)

**Comment by forman** [comment]
Use this

```yaml
process:
  rechunk:
    "*":
      lon: 2880
      lat: 1440
      time: 1
    lon: null
    lat: null
    time: 365
    time_bnds:
      time: 365
      bnds: 2
```

**Comment by forman** [comment]
Will be closed by #15
